src/main/java/io/snowdrop/github/reporting/IssueCollector.java

```diff
@@ -68,38 +72,44 @@ public void refresh() {
   }
 
   public Map<String, Set<Issue>> collectIssues() {
+    init(Optional.empty(), Optional.empty());
```

Comment:
It's not clear to me why we pass empty and not reportingDay, reportingTime?

Reply:
Si the're obtained from the system date.

Reply:
Still dont understand this.